src/pages/patientView/PatientViewPage.tsx

```diff
@@ -38,10 +38,16 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             );
         }
 
+        let samples = ["string", "TCGA-24-2024-01", "TCGA-24-2024-01 (deliberately elongated id)"];
         let mutationDiv: Element | null = document.getElementById('mutations_div_prototype');
         if (mutationDiv) {
             ReactDOM.render(
-                <MutationInformationContainer {...{store: this.props.store}} />,
+                <MutationInformationContainer sampleOrder={samples}
+                                             sampleLabels={samples.reduce((map, sample, i) => {map[sample] = i+""; return map;}, {})}
+                                            sampleColors={samples.reduce((map, sample) => {map[sample] = (Math.random() > 0.5 ? "black": "red"); return map;}, {})}
+                                              sampleTumorType={samples.reduce((map, sample) => {map[sample] = (Math.random() > 0.5 ? "Primary": "Metastasis"); return map;},{})}
+                                              sampleCancerType={samples.reduce((map, sample) => {map[sample] = "Cancer Type of " + sample; return map;}, {})}
+                                            {...{store: this.props.store}} />,
```

Comment:
I think that it's natural to pass it in from `PatientViewPage` since that's where the context is of the relevant samples. Whether it should be a required prop for `MutationInformationContainer` is another story.